### ISSUE TITLE
[📝 Docs]: Discussion on Feedback Form Usage and Increasing Conversion

### ISSUE THREAD

**Issue by DarhkVoyd** [json-schema-org/website#762]
### What Docs changes are you proposing?

**Description:**
We have identified a recurring pattern where users are submitting questions through the feedback form on the website. But, users are not following up by raising an issue or joining the Slack server for further discussion. We do not take email or name, as this isn't a support form. Any ideas on how we can increase conversion on this?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by DarhkVoyd** [comment]
**Suggestions:**
   - Make it clear at the beginning of the form that it is intended for feedback, not for support or questions.
   - Include a message in the form confirmation reiterating that the feedback form is for feedback only. For questions, direct users to join the Slack server for discussion.
   - Modify the current form response to clearly inform users that their feedback will be published in the Slack channel #website for further discussion.

**Comment by benjagm** [comment]
What I think we can easily do is clarify muy better the feedback alternatives we provide:
* To provide feedback and engage in discussion with us the should use the create issue button.
* To provide anonymous feedback they should use the submit feedback button.

![Screenshot 2024-06-19 at 10 24 46](https://github.com/json-schema-org/website/assets/40007659/917d36c8-d64d-4d7f-97df-921dd7afa259)

**Comment by gregsdennis** [comment]
I think instead of "submit feedback" we should use something more explicit, like "Yeah I'm not looking for a response."

Seriously though, if people are asking a question, then the assumption is they want an answer, not for the page to just update.  Many of the questions we've received don't have answers that necessarily fit on the page where they submitted it.

Personally, it's frustrating when a user has a question and I have an answer but I can't provide it because they just clicked "submit feedback" without actually looking at their options. 

---

What I'd like to see is a form showing fields for all the contact details that we'd need to reply and a pre-checked checkbox with "I'd like a reply."  Then the contact fields are required as long as the checkbox is checked.

Unchecking is opting out, and they don't need to provide contact details. 

The opt-out model is the approach I typically see websites use, but the checkbox is usually something like "I want to receive the newsletter" or "I want to receive offers".

**Comment by benjagm** [comment]
> What I'd like to see is a form showing fields for all the contact details

I am totally agains converting this into a contact form. Again, I think your as speaking amore about your feelings because of not being able to reply, rather than the actually feedback we get.

You said we get mostly questions, but that is not accurate:
- So far we received 98 replies. 74 positive and 24 negative. 
- From the 24 negative, only 6 are questions and most seems rhetorical questions to express an opinion.

My point is, this is all about the user, not us and how do we feel with the comments. The bast path forward is to turn all this feedback into actionable improvements and improve our docs the best way we can and later communicate it.

Additionally we'll make sure we explain better the options to transparent feedback.

This article express my approach here:
https://customerthink.com/what-are-the-benefits-of-anonymous-customer-feedback/

**Comment by gregsdennis** [comment]
> turn all this feedback into actionable improvements

This can't be done with the ones I'm talking about.  The one that sparked this entire debate was a question about how to use a specific implementation.  We can't include that in our docs or tools list.

**Comment by benjagm** [comment]
> This can't be done with the ones I'm talking about.

I strongly disagree.

If this user started its JSON Schema journey in the getting started page which focus is to teach you how to create a schema and validate it, but they were looking for a schema generation tutorial and ended up doing this question, we can improve this page to better explain that there are other uses cases and where to find answers for those questions and making in clear that we are implementation agnostic.... so we don promote any tool over others.

**Comment by benjagm** [comment]
I think everything is a matter of balance between individual support and broader feedback. You seems to be focused on individual support which is great, while I am focused on capturing as much feedback as possible to turn it into improvements and help the broader community.

As I said it is a matter of balance and I think we can do it without having to create a contact form. We'd like the discussion to happen in the community and not in an email thread.

I have just made some changes in the component to prioritize the feedback section so the first question available is "Do you have a question?". The next option available will be Feedback ... se we are explicitly presenting the user to option to ask a question before anything else.

My proposal:
![Screenshot 2024-06-19 at 14 03 16](https://github.com/json-schema-org/website/assets/40007659/22712020-4609-4af9-9415-1ef628d50aa5)

Current version:
![Screenshot 2024-06-19 at 14 03 58](https://github.com/json-schema-org/website/assets/40007659/cd4d0f38-466a-4883-9636-06004d3f4c4d)

With this proposal we are prioritizing the option to get support and make a question to the Community. If this works we'll reduce the number of questions received in the feedback form.

**Comment by Relequestual** [comment]
If we might fix this problem simply be re-ordering the footer, that feels like a simple win.

Although, maybe we could change "often confusing" to "sometimes confusing"? 😅

**Comment by gregsdennis** [comment]
> We'd like the discussion to happen in the community and not in an email thread.

I think this may be the disconnect.  DV's solution in slack was to create an issue or discussion in GitHub and respond there.  No one is suggesting we respond via email.  At most, they get an email with a link to the issue/discussion that was created so they know where to find a reply.

I'm also not saying we should be collecting people's data.  We absolutely should not be doing that.

I still think we need to prioritize providing support.

**Comment by gregsdennis** [comment]
Let start with your suggestion @benjagm.  See if that improves things.

**Comment by gregsdennis** [comment]
This is still very much needed.

**Comment by benjagm** [comment]
> This is still very much needed.

Agreed.

**Comment by harshsbhat** [comment]
@benjagm maybe #841 that I created tries to fix something.

**Comment by jagpreetrahi** [comment]
Hey @benjagm  , what if we collect minimal optional data e.g. feedback type  & topic  , to tailor the responses and suggest relevant resources.
or even we can suggest FAQs, articles, or GitHub issues dynamically based on the user’s input.

I think it would be user-centric experience , if we think about it.

**Comment by idanidan29** [comment]
I agree with @benjagm’s proposal it makes a lot of sense, In addition I suggest adding a text input below the quick feedback options to allow users to describe their feedback, making it more engaging and more easy for us to improve from it. 

for support i really like @jagpreetrahi idea, linking to an easy to navigate FAQ section or dynamically suggesting relevant FAQs based on user input would really improve the user experience and help users find answers faster.

**Comment by Kashika23** [comment]
@benjagm can i work on it further.

**Comment by Mounika-max-348** [comment]
@benjagm can i work for this issue.

**Comment by manya1632** [comment]
Hi @benjagm  @DarhkVoyd  !
I’d like to work on this issue if it’s still available.
I plan to start by implementing the intent-first change discussed above (prioritizing “question vs feedback” in the form) and improving the copy to clearly redirect questions to GitHub/Slack.

Assign it to me, I'll make a PR asap and feel free to drop any changes you'd like to suggest.

**Comment by Vedantu2005** [comment]
Hi everyone 👋
I’d like to contribute to this issue.
I’m planning to implement an intent-first feedback form with:

Clear “Question vs Feedback” choice
Redirect questions to GitHub Discussions
Improved confirmation message

Please let me know if this approach aligns with the current direction.
